components/calendar/src/julian.rs

```diff
@@ -0,0 +1,198 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+//! This module contains types and implementations for the Julian calendar
+
+#[derive(Copy, Clone, Debug, Default)]
+// The Julian calendar
+pub struct Julian;
+
+#[derive(Copy, Clone, Debug, Hash, Eq, PartialEq)]
+// The inner date type used for representing Date<Georgian>
+pub struct JulianDateInner(IsoDateInner);
+
+impl Calendar for Julian {
+    type DateInner = JulianDateInner;
+    fn date_from_iso(&self, iso: Date<Iso>) -> JulianDateInner {
+        *iso.inner()
```

Comment:
issue: this and the next function will need to convert: you're using IsoDateInner for convenience here but it's worth noting that the inner value is not actually an ISO compatible date.